src/locale/locales/tr/messages.po

```diff
@@ -4572,7 +4572,7 @@ msgstr "Listeyi sessize al"
 #: src/components/dms/ConvoMenu.tsx:136
 #: src/components/dms/ConvoMenu.tsx:142
 #~ msgid "Mute notifications"
-#~ msgstr ""
+#~ msgstr "Bildirimleri sustur"
```

Comment:
For the native turkish speakers:
`Bildirimleri sustur` ve `Bildirimleri sessize al` arasında kararsızım. Bir hesap için kullanılan mute ile bildirimler için kullanılan mute aynı anlamda mı?

Reply:
Bildirimleri Sessize Al daha kulağa aşina

Reply:
Ben dahil `Sessize Al` için 3 adet oyumuz var. Yaklaşık 9 ay önce ilk çeviriyi yapan @atiksoftware'i de bu konuşmaya davet etmek istiyorum. Nasıl bir yol izlenmeli?

Ben, öncesinde hep `Sustur` şeklinde çevrildiğinden tutarlı olması için sustur diye eklemiştim.